packages/discord.js-utilities/src/lib/PaginatedMessage.ts

```diff
@@ -166,37 +176,18 @@ export class PaginatedMessage {
 
 		const firstPage = this.messages[this.index]!;
 
-		const response = (await channel.send(firstPage)) as Message;
+		this.response ??= (await channel.send(firstPage)) as Message;
```

Comment:
Shouldn't this override instead? Since, you could possibly call `run()` with a new author, or new channel.